content/06.model-complexity.md

```diff
@@ -38,4 +38,4 @@ In a study using a variational autoencoder (VAE) (see Box 2: Definitions) for di
 In a study using a convolutional neural network (CNN) to identify tubers in MRI images from tuberous sclerosis patients, overfitting was minimized using the dropout regularization method which removed randomly chosen network nodes in each iteration of the CNN model generating simpler models in each iteration.[@doi:10.1371/journal.pone.0232376]
 Thus, depending on the learning method that is used, regularization approaches should be incorporated into data analysis when working with rare disease datasets. 
 
-![TODO: OLD FIGURE (new figure still WIP) Strategies to simplify models and stabilize predictions preserve the value of machine learning in rare disease. A-B) Strategies to build confidence in model predictions; A) A schematic showing the concept of bootstrap, B) A schematic showing the concept of ensemble learning to converge on reliable models; C-D) Strategies to simplify models by penalizing complexity in ML models; C) A schematic showing the concept of regularization to selectively learn relevant features, D) A schematic showing the concept of one-class-at-a-time learning to select few features at a time. Horizontal bars represent health of a model, models are represented as a network of nodes (features) and edges (relationships), nodes with solid edges represent real patterns, nodes with broken edges represent spurious patterns](images/figures/pdfs/statistical-techniques.png){#fig:3}
+![Strategies to reduce misinterpretation of machine learning model output in rare disease. a) Bootstrapping: Left panel shows a small rare disease dataset, which can be resampled with replacement using bootstrap to form a large resampled dataset (middle panel). Running the same ML model on multiple resampled datasets generates a distribution of values for the importance scores for each feature utilized by the ML model (right panel), b) Cascade Learning: A schematic showing the different steps in a cascade learning approach for identifying rare disease patients from electronic health record data[@pmid:30815073]. c) Regularization: A schematic showing the concept of regularization to selectively learn relevant features. The samples (green and blue circles) in the rare disease dataset on left panel can be represented as a combination of features. Each horizontal bar in the middle panel (training set) represents a feature X sample heatmap for one sample each. The dark and light patches show the value of the features in representing the sample denoted by the colored sample next to each bar. In the held-out validation dataset, for a sample of unknown class (open circle), some features recapitulate the pattern present in training set, while others do not. The right panel shows classification accuracy with and without using regularization during implementation of the ML models on rare disease data.](images/figures/pdfs/figure-3.png){#fig:3}
```

Comment:
I would say what the outcome of cascade learning is maybe? Accuracy goes up, error goes down...

Reply:
Also consider expanding on this point.

> The right panel shows classification accuracy with and without using regularization during implementation of the ML models on rare disease data.

 I know that you say "to selectively learn features" in the first sentence describing this panel, but that strikes me as potentially a little bit jargon-y.